dev-python/urllib3/urllib3-1.26.4.ebuild

```diff
@@ -58,6 +58,35 @@ python_test() {
 	local deselect=(
 		# TODO?
 		test/with_dummyserver/test_socketlevel.py::TestSocketClosing::test_timeout_errors_cause_retries
+		# Failed: Timeout >60.0s
```

Comment:
I'm sorry but this is a big *no* from me. There is clearly some breakage involved here, and I would really like to have upstream at least look into it first.

Reply:
Seems like they already know about it: https://github.com/urllib3/urllib3/pull/2209. So we'll just have to wait until this is fixed, I guess..

Reply:
Alright, I applied the patch mentioned there and the tests mentioned are now passing.